migrations/20171211_sanitize_emails.js

```diff
@@ -0,0 +1,14 @@
+/*
+  User creation saves email as lowercase, but updating an email did not.
+  Run this script to ensure all lowercased emails in db AFTER fix for updating emails is implemented.
+  This will fix inconsistent querying for an email when attempting to password reset.
+*/
+
+db.users.find({'auth.local.email':{$not: /[A-Z]/}}).forEach(user => {
```

Comment:
Will this actually find addresses that have no capital letters?

Reply:
I fixed the query in order to find all emails that have any capital letters.